phpstan.neon

```diff
@@ -0,0 +1,9 @@
+parameters:
+    level: 8
+    paths:
+        - src/
+    bootstrapFiles:
+        - tests/bootstrap.php
+    checkGenericClassInNonGenericObjectType: false
```

Comment:
Please fix according to https://spryker.slack.com/archives/C7N7YEHK8/p1650969879005379